wiring/inc/spark_wiring_cellular.h

```diff
@@ -50,6 +50,12 @@ class CellularClass : public NetworkClass
     void off() {
         network_off(*this, 0, 0, NULL);
     }
+    bool isOn() {
```

Comment:
Nit: could potentially just add to `NetworkClass`. Yeah, it's a bit messy right now, but this should be generic and available in the parent class.

Reply:
Will remove these from spark_wiring_cellular.h and spark_wiring_wifi.h